website/views/shared/modals/level-up.jade

```diff
@@ -16,6 +16,26 @@ script(type='text/ng-template', id='modals/levelUp.html')
       h4(style='margin-top: 1em')!=env.t('leveledUp', {level:'{{user.stats.lvl}}'})
       p=env.t('fullyHealed')
       br
+      div(ng-if='user.flags.classSelected && !user.preferences.disableClasses && !user.preferences.automaticAllocation')
+        a.btn.btn-default(data-toggle='collapse', data-target='#stat-allocation', aria-expanded=false, aria-controls='stat-allocation')=env.t('showHideQuickAllocation')
```

Comment:
Can the "Show/hide" text change depending on what's being shown, so that it says either "show" or "hide"?